libc/string/string_lenchr.c

```diff
@@ -409,6 +568,17 @@ TEST_GROUP_RUNNER(string_len)
 }
 
 
+TEST_GROUP_RUNNER(string_spn)
+{
+	RUN_TEST_CASE(string_spn, ascii);
+	RUN_TEST_CASE(string_spn, not_ascii);
+	RUN_TEST_CASE(string_spn, empty_output);
+	RUN_TEST_CASE(string_spn, mixed_order);
+	RUN_TEST_CASE(string_spn, basic);
+	RUN_TEST_CASE(string_spn, big);
```

Comment:
The basic case should always be tested at the beginning.

```suggestion
	RUN_TEST_CASE(string_spn, basic);
	RUN_TEST_CASE(string_spn, ascii);
	RUN_TEST_CASE(string_spn, not_ascii);
	RUN_TEST_CASE(string_spn, empty_output);
	RUN_TEST_CASE(string_spn, mixed_order);
	RUN_TEST_CASE(string_spn, big);
```